src/Template/Adrs/xml/e2b.ctp

```diff
@@ -252,7 +256,7 @@
                 <reactiongestationperiod/>
                 <reactiongestationperiodunit/>
                 <drugindicationmeddraversion/>
-                <drugindication><?php echo $adrListOfDrug['indication']; ?></drugindication>
+                <drugindication><?php echo preg_replace('/[^A-Za-z0-9\s+]/', '',$adrListOfDrug['indication']); ?></drugindication>
```

Comment:
Create a utility method that will perform the replace function and reuse it as appropriate